src/widgets/Window.cpp

```diff
@@ -71,6 +71,7 @@ Window::Window(WindowType type, QWidget *parent)
             this->onAccountSelected();
         }));
     this->onAccountSelected();
+    this->setAttribute(Qt::WA_TranslucentBackground);
```

Comment:
Could we only set this if we know the color we will paint with is actually transparent? I suppose that would be `getTheme()->windows->background`. Then it should probably be set in `BaseWindow` instead, since it paints with this color.

Reply:
I don't think the transparency should appear on all windows. Only the main one is preferable (`BaseWindow` doesn't know what a `Window::Main` is), I'll make the window check for the theme being transparent before setting the flag.